src/js/components/Layer/Layer.js

```diff
@@ -54,10 +54,9 @@ const Layer = forwardRef((props, ref) => {
           }
           setTimeout(() => {
             // we add the id and query here so the unit tests work
-            const clone =
-              containerTarget === document.body
-                ? document.getElementById('layerClone')
-                : containerTarget.getElementById('layerClone');
+            const clone = containerTarget
+              .getRootNode()
```

Comment:
Will this ever be undefined?
```suggestion
              .getRootNode()?
```

Reply:
I don't think so: https://developer.mozilla.org/en-US/docs/Web/API/Node/getRootNode#return_value